File: 0217-Contains-duplicates/0217-contains-duplicates.py
import numpy
from collections import Counter
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        result = []
        np_array = numpy.array(nums)
        result = [item for item, count in Counter(np_array).items() if (count > 1)]
        if (result == []):
            return False
        else:
            return True

File: 0217-Contains-duplicates/0217-Contains-duplicates.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let a = 0;
    for (i in nums) {
        a = nums[i];
        var tmp = nums.filter(value => value == a);
        if (tmp.length > 1)
            return true;
    }
    return false;
};
